openlibrary/templates/account/create.html

```diff
@@ -8,76 +10,79 @@
-
-        <label>
-            $:form.ia_newsletter.render() $:form.ia_newsletter.description
-        </label>
-
-        <hr/>
-
-        <small>$:_("By signing up, you agree to the Internet Archive's <a href='//archive.org/about/terms.php' target='_blank'>Terms of Service</a>.")</small>
 
-        <div class="formElement bottom">
-            <br/>
-            $ sitekey = classes = callback = ''
-            $if form.has_recaptcha:
-                $ callback = "data-callback=submitCreateAccountForm"
-                $ classes = 'g-recaptcha'
-                $ sitekey = "data-sitekey=%s" % form['recaptcha'].public_key
-                $:render_template("recaptcha", form['recaptcha'].public_key, error=None, invisible=True)
-            <button type="submit" name="signup" id="signup" class="larger $classes" $sitekey $callback>$_("Sign Up")</button>
-            <a href="javascript:history.go(-1);" class="smaller attn">$_("Cancel")</a>
-        </div>
-    </form>
+    $if ctx.user:
+        $def user_link(): <a href="$ctx.user.key">$ctx.user.displayname</a>
+        <p>$:_("You are already logged into Open Library as %(user)s.", user=str(user_link()))</p>
+        <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'hamburger-logout\'].submit()">log out</a> and start the signup process afresh.')</p>
+    $else:
+        <form class="olform create create-form" name="signup" method="post" data-i18n="$json_encode(i18n_strings)" action="">
+            $if form.note:
+                <div class="note">$form.note</div>
+
+            $def screenname_url(): $_('Your URL'): https://openlibrary.org/people/<span id="userUrl">$(form.username.value or _('screenname'))</span>
+
+            $:field(form.email)
+            $:field(form.username, suffix=str(screenname_url()))
+            $:field(form.password)
+
+            <div class="ol-signup-form__newsletter">
+                $:form.ia_newsletter.render() <label for="ia_newsletter">$:form.ia_newsletter.description</label>
+            </div>
+
+            <div class="formElement bottom">
+                <br/>
+                $ sitekey = classes = callback = ''
+                $if form.has_recaptcha:
+                    $ callback = "data-callback=submitCreateAccountForm"
+                    $ classes = 'g-recaptcha'
+                    $ sitekey = "data-sitekey=%s" % form['recaptcha'].public_key
+                    $:render_template("recaptcha", form['recaptcha'].public_key, error=None, invisible=True)
+                <button type="submit" name="signup" id="signup" class="larger $classes cta-btn cta-btn--primary" $sitekey $callback>$_("Sign Up with Email")</button>
+                <p class="ol-signup-form__tos sansserif lighter">$:_('By signing up, you agree to the Internet Archive\'s <a class="ol-signup-form__link--muted" href="//archive.org/about/terms.php" target="_blank">Terms of Service</a>.')</p>
```

Comment:
This makes more sense in a `<small>` take for semantic html

```suggestion
                <small class="ol-signup-form__tos sansserif lighter">$:_('By signing up, you agree to the Internet Archive\'s <a class="ol-signup-form__link--muted" href="//archive.org/about/terms.php" target="_blank">Terms of Service</a>.')</small>
```